src/main/java/org/blackdread/sqltojava/entity/JdlFieldEnum.java

```diff
@@ -26,7 +26,8 @@
     /**
      * Defined here to allow to have a pattern set for TIME type of SQL that jdl does not support by default
      */
-    TIME_AS_TEXT;
+    TIME_AS_TEXT,
+    GEOMETRY;
```

Comment:
I see added but supported by the JDL ? -> I see no particular export logic code yet with this type

Reply:
Have changed it from GEOMETRY to GEOMETRY_AS_TEXT so which seems to be the common workaround that others use to re-gen their applications